api/Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN pip install poetry poetry-plugin-export
 
 WORKDIR /home/pcapi
 
+RUN poetry config warnings.export false
```

Comment:
[détail / discussion ] J'ai lu l'issue que tu mentionnes, c'est un peu moche... Peut-être qu'on pourrait au moins ajouter un FIXME qui explique pourquoi on a besoin de cette commande et quand on pourra la supprimer ? Suggestion : 

```dockerfile
# FIXME: this is to avoid a warning about "poetry-plugin-export" not being installed,
# even if it is installed. That line can be removed once Poetry does not warn us anymore.
```

Reply:
c'est fait, merci !